Makefile

```diff
@@ -73,17 +73,19 @@ run: build-local-container kind-load deploy ## Build image and run operator in-c
 
 # Binary builds
 GO_BUILD := $(Q)go build
+GO_OS=$(if $(GOOS),GOOS=$(GOOS))
```

Comment:
I had to use this extra variable to get the behavior that was described below. Simply setting `GOOS=linux` as an environmental variable to the build-local-container target did not result in a linux binary being built for whatever reason.

Reply:
Hmm I swear I made that change locally and had it working. Let me try something.